Leanplum/Cache/LPCache.h

```diff
@@ -0,0 +1,20 @@
+//
+//  LPCache.h
+//  Leanplum
+//
+//  Created by Grace on 7/23/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "LPCacheProtocol.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LPCache : NSObject<LPCacheProtocol>
```

Comment:
We can temporarily name it `LPCache`, but I envision this being called `LPSQLiteCache` or `LPUserDefaultsCache`, as the case may be.